thunder/tests/opinfos.py

```diff
@@ -2232,6 +2232,14 @@ def fmod_sample_input_generator(op, device, dtype, requires_grad, **kwargs):
 )
 elementwise_binary_ops.append(logical_and_opinfo)
 
+logical_xor_opinfo = OpInfo(
+    clang.logical_xor,
+    dtypes=(datatypes.exact, datatypes.floating),
```

Comment:
Why are complex dtypes excluded? Or rather, what fails if this line is deleted?

(I missed it in the first pass over #1842 but the same question holds for that opinfo.)